google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/CreateTopicAndPublishMessages.java

```diff
@@ -78,6 +80,54 @@ public static void publishMessages() throws Exception {
     // [END pubsub_publish]
   }
 
+  public static void publishMessagesWithErrorHandler() throws Exception {
+    // [START pubsub_publish_error_handler]
+    ProjectTopicName topicName = ProjectTopicName.of("my-project-id", "my-topic-id");
+    Publisher publisher = null;
+
+    try {
+      // Create a publisher instance with default settings bound to the topic
+      publisher = Publisher.newBuilder(topicName).build();
```

Comment:
Should we use the try-with-resource here?

Reply:
It seems that Publisher hasn't implemented java.lang.AutoClosable :(

Reply:
You're right. The shutdown is longrunning so it's not closeable in the "normal" sense.